packages/mobile/src/identity/actions.ts

```diff
@@ -312,6 +320,24 @@ export const updateE164PhoneNumberAddresses = (
   addressToE164Number,
 })
 
+export const updateWalletToAccountAddress = (
+  walletToAccountAddress: WalletToAccountAddressType
+): UpdateWalletToAccountAddressAction => {
+  const newWalletToAccountAddresses: WalletToAccountAddressType = {}
+  const walletAddresses = Object.keys(walletToAccountAddress)
+
+  for (const walletAddress of walletAddresses) {
+    const newWalletAddress = walletAddress.toLowerCase()
```

Comment:
Similar to using `eqAddress` elsewhere, we likely want to use `normalizeAddress` from `@celo/base` here and anywhere else where our goal is to normalize the address.